### PR TITLE
Update and delete enterprise

### PR DESCRIPTION
**Pull request by hawkesky**

### CODE REVIEW

File: app/tests/test_enterprise.py [hunk 1]
```diff
@@ -53,3 +53,20 @@ def test_enterprise_get_for_user(client: TestClient):
...
+        json={"name": "Zdzichtex sp. z o.o."})
+    assert r_update.status_code == HTTP_200_OK
+
+def test_delete_enterprise(client: TestClient):
+    user_header = get_random_logged_user(client)
+    r_enterprise = create_enterprise(client)
+    r_enterprise_delete = client.delete(
+        f"/enterprise/{r_enterprise.json()['id']}",
+        cookies=client.cookies.get_dict())
+    assert r_enterprise_delete.status_code == HTTP_200_OK
```
Comment (eredzik) [inline app/tests/test_enterprise.py:72]: Add assertion that enterprise is in fact not existing (try using get on it)

File: app/routes/enterprise.py [hunk 2]
```diff
@@ -89,4 +99,65 @@ async def create_enterprise(
...
+
+        permissions = await verify_enterprise_permissions(
+            user,
+            enterprise_id,
+            required_permissions=[
+                models.UserEnterpriseRoles.admin,
+            ],
+        )
+        if permissions is True:
+            enterprise.delete()
```
Comment (eredzik) [inline app/routes/enterprise.py:159]: Doesnt it need await?

File: app/routes/enterprise.py [hunk 3]
```diff
@@ -89,4 +99,65 @@ async def create_enterprise(
...
+                role="ADMIN",
+                nip_number=enterprise.nip_number,
+                address=enterprise.address,
+            )
+            return enterprise_output
+        return permissions
+
+    @enterprise_router.delete(
+        "/enterprise/{enterprise_id}",
+        status_code=200,
```
Comment (eredzik) [inline app/routes/enterprise.py:139]: It needs response model - frontend build fails if there is no response model

File: app/routes/enterprise.py (hunk 2 above)
Comment (hawkesky) [reply app/routes/enterprise.py:159]: I think it doesn't, as the content of the "enterprise" variable already gets await

File: app/routes/enterprise.py (hunk 3 above)
Comment (hawkesky) [reply app/routes/enterprise.py:139]: Fair enough, I'll apply the same to invoice

**Pull request merged**